frontend/tests/controllers/ProblemCreateTest.php

```diff
@@ -519,6 +519,51 @@ public function testCreateProblemWithTags() {
         );
     }
 
+    /**
+     * test for count problems whit levelTag
+     */
+    public function testCountProblemsWithLevelTags() {
+        // Get the problem data
+        $problemData = \OmegaUp\Test\Factories\Problem::getRequest();
+        $r = $problemData['request'];
+        $problemAuthor = $problemData['author'];
+
+        // Login user
+        $login = self::login($problemAuthor);
+        $r['auth_token'] = $login->auth_token;
+
+        // Create problems by level
+        $problemLevelMapping = [
+            'problemLevelBasicIntroductionToProgramming' => 5,
+            'problemLevelIntermediateMathsInProgramming' => 5,
+            'problemLevelIntermediateDataStructuresAndAlgorithms' => 5,
+            'problemLevelIntermediateAnalysisAndDesignOfAlgorithms' => 5,
+            'problemLevelAdvancedCompetitiveProgramming' => 5,
+            'problemLevelAdvancedSpecializedTopics' => 5,
+            'problemLevelBasicKarel' => 5,
+        ];
+        $problemData = [];
+        foreach ($problemLevelMapping as $level => $numberOfProblems) {
+            foreach (range(0, $numberOfProblems - 1) as $_) {
+                $problemData[] = \OmegaUp\Test\Factories\Problem::createProblem(
+                    new \OmegaUp\Test\Factories\ProblemParams([
+                        'problem_level' => $level,
+                    ])
+                );
+            }
+        }
+
+        $problemsCount = [];
+        $total = 0;
+        $response = \OmegaUp\Controllers\Problem::getProblemCollectionDetailsForSmarty()['smartyProperties']['payload'];
```

Comment:
Todas las funciones que terminan con `ForSmarty` reciben como parámetro `\OmegaUp\Request`, en este caso que no se requiere que exista un usuario logueado lo puedes mandar de esta forma:
```suggestion
        $response = \OmegaUp\Controllers\Problem::getProblemCollectionDetailsForSmarty(
            new \OmegaUp\Request([])
        )['smartyProperties']['payload'];
```
?